docs/interplanetary-file-system.md

```diff
@@ -0,0 +1,67 @@
+
+- It is not necessary to know the address, only what is needed is searched in the system.
+- The system sends the search that is being carried out to the network of nodes. This information is protected by encryption, hash and digital signature, this in order to protect the information and give access only to those who are authorized.
+- After sending the search, the nodes respond with information on the available versions of what was requested.
+- By choosing one of the results obtained, you can see both its content and its information history.
+
+That is the way the IPFS system works, for this reason it is known as **search by content** since we simply indicate what we need and the network of nodes responds with the option to consult the data that is going to be required.
+
+## Advantages and disadvantages of IPFS
+
+Some of the advantages are as follows:
+
+- Decentralized storage so it does not depend on a single place for access and is accessible at any time.
+- It is scalable, so future improvements and functionalities can be added.
+- Fast data storage and query.
+- Greater storage capacity.
+- There is no censorship of content, being decentralized it is impossible for any content to be censored.
+
+Some of its disadvantages:
+- It is a new technology that when implemented will bring with it some complications when wanting to run it completely on the internet.
+- It is still in the process of improvement and development.
+- Being new is difficult to use for some people.
+
+## IPFS Use Cases
+
+Currently there are several cases of use of systems in which companies have decided to innovate using this new technology. This is due to the large number of benefits offered by this system, some companies that use this technology are: [Audius](https://audius.co/), a platform that uses IPFS technology to store your data in a decentralized way where the artist can see who listens to their music so they have total control over it, another well-known company that is starting to use this technology to manage their files is [Netflix](https://www.netflix.com/).
+
+This IPFS protocol has also been implemented in various networks related to the blockchain. This is due to its storage capacity and large number of benefits that it can bring to these networks. Some of the networks that use this storage protocol are: [Filecon](https://filecoin.io/), it is a decentralized storage network, the first blockchain network to implement the IPFS protocol in its network, [Storj](https://www.storj.io/), is another decentralized storage blockchain network that also uses the protocol. Another great use case is with NFTs ([Non Fungible Tokens](https://guias.eoscostarica.io/docs/aprender-eosio/nfts-en-eosio)), this technology is currently trending and is used for encrypted digital assets which uses IPFS for the storage of its data.
```

Comment:
Remove "This"

Reply:
The NFT link is referencing the Spanish site , please point to the english equivalent

Reply:
Attended.